### PR TITLE
Add more primereact facades

### PR DESCRIPTION
**Pull request by toddburnside**
This contains most of the form sort of controls that we are using in Explore.

Raul is using Accordion in Observe, and I will do that next in another PR. It has some intricacies to make it safe to use.

### CODE REVIEW

File: prime-react/src/main/scala/react/primereact/Dropdown.scala [hunk 1]
```diff
@@ -3,40 +3,38 @@
...
-  showClear:       js.UndefOr[Boolean] = js.undefined,
   filter:          js.UndefOr[Boolean] = js.undefined,
   showFilterClear: js.UndefOr[Boolean] = js.undefined,
   placeholder:     js.UndefOr[String] = js.undefined,
   disabled:        js.UndefOr[Boolean] = js.undefined,
-  onChange:        js.UndefOr[Any => Callback] = js.undefined
-) extends ReactFnProps[Dropdown](Dropdown.component)
+  dropdownIcon:    js.UndefOr[String] = js.undefined,
+  onChange:        js.UndefOr[A => Callback] = js.undefined
+)(using val eqAA:  Eq[A])
```
Comment (cquiroz) [inline prime-react/src/main/scala/react/primereact/Dropdown.scala:27]: Nice

File: prime-react/src/main/scala/react/primereact/InputTextarea.scala [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  autoResize:  js.UndefOr[Boolean] = js.undefined,
+  disabled:    js.UndefOr[Boolean] = js.undefined,
+  placeholder: js.UndefOr[String] = js.undefined,
+  rows:        js.UndefOr[Int] = js.undefined,
+  onBlur:      js.UndefOr[ReactFocusEventFromTextArea => Callback] = js.undefined,
+  onChange:    js.UndefOr[ReactEventFromTextArea => Callback] = js.undefined,
+  onKeyDown:   js.UndefOr[ReactKeyboardEventFromTextArea => Callback] = js.undefined
+) extends ReactFnProps[InputTextarea](InputTextarea.component)
+
+object InputTextarea {
```
Comment (cquiroz) [inline prime-react/src/main/scala/react/primereact/InputTextarea.scala:26]: Should tihs be `InputTextArea`?

**Review by cquiroz — APPROVED** [review]
Great work, thanks

**Comment by rpiaggio** [comment]
Thank you very much for doing this.

The API is looking great.

I'm a bit concerned by the extra layer of components. I don't know if this has a noticeable impact on the artifact size or on runtime performance. The alternative is go for a fully manually defined facade similar to the one for SUI. Any thoughts?

**Comment by rpiaggio** [comment]
> My thoughts were that we would start using it and tweak as needed, then decide if/when we want to make it manual.

Sounds good :+1:

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**